src/views/FileListInfo.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+	<div v-if="active" class="filelist-collectives-wrapper">
+		<div class="infobox">
+			<InformationIcon fill-color="var(--color-primary-element)" />
+
+			<div class="content">
+				{{ t('collectives', 'The content of this folder is best viewed in the Collectives app.') }}
+
+				<a :href="collectivesLink">
+					<NcButton :aria-label="t('collectives', 'Open in Collectives')"
+						type="primary"
+						class="button">
+						{{ t('collectives', 'Open in Collectives') }}
+					</NcButton>
+				</a>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import { generateUrl } from '@nextcloud/router'
+import { NcButton } from '@nextcloud/vue'
+import InformationIcon from 'vue-material-design-icons/Information.vue'
+import { GET_COLLECTIVES_FOLDER } from '../store/actions.js'
+import { mapActions, mapState } from 'vuex'
+import displayError from '../util/displayError.js'
+
+export default {
+	name: 'FileListInfo',
+
+	components: {
+		InformationIcon,
+		NcButton,
+	},
+
+	props: {
+		path: {
+			type: String,
+			required: true,
+		},
+	},
+
+	data() {
+		return {
+			active: false,
+		}
+	},
+
+	computed: {
+		...mapState({
+			collectivesFolder: (state) => state.settings.collectivesFolder,
+		}),
+
+		collectivesLink() {
+			return generateUrl('/apps/collectives' + this.path.replace(this.collectivesFolder, ''))
```

Comment:
Maybe check if the string starts with and slice it then to avoid issues when the collectivesFolder name is used somewhere in the page path

Reply:
Good catch, fixed now :+1: